<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocoEditForm</class>
 <widget class="QDialog" name="LocoEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>326</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>445</width>
    <height>326</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>445</width>
    <height>326</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Редактирование данных локомотива</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="general_verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="loco_type_horizontalLayout">
       <item>
        <widget class="QLabel" name="loco_type_label">
         <property name="text">
          <string>Серия локомотива</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="loco_type_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="loco_type_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="thurst_force_horizontalLayout">
       <item>
        <widget class="QLabel" name="thurst_force_label">
         <property name="text">
          <string>Расчетная сила тяги</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="thurst_force_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="thrust_force_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="mass_horizontalLayout">
       <item>
        <widget class="QLabel" name="mass_label">
         <property name="text">
          <string>Расчетная масса</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="mass_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="mass_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="constr_valocity_horizontalLayout">
       <item>
        <widget class="QLabel" name="constr_velocity_label">
         <property name="text">
          <string>Конструкционная скорость</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="constr_valocity_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="constr_velocity_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="calc_velocity_horizontalLayout">
       <item>
        <widget class="QLabel" name="calc_velocity_label">
         <property name="text">
          <string>Расчетная скорость</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="calc_velocity_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="calc_velocity_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="length_horizontalLayout">
       <item>
        <widget class="QLabel" name="length_label">
         <property name="text">
          <string>Длина</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="length_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="length_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="traction_horizontalLayout">
       <item>
        <widget class="QLabel" name="traction_label">
         <property name="text">
          <string>Тяговая характеристика</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="traction_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="traction_pushButton">
         <property name="text">
          <string>Загрузить из Excel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="traction_lineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttons_horizontalLayout">
       <item>
        <widget class="QPushButton" name="delete_pushButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="buttons_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LocoEditForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LocoEditForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
